charts/br-sta/values.yaml

```
@@ -0,0 +1,666 @@
+      # Operators MUST supply a stable cookie (e.g. openssl rand -base64 32) when
+      # rabbitmq.enabled=true. It must not change across upgrades (breaks clustering).
+      value: ""
+
+# ==============================================================================
+# OPTIONAL OTEL COLLECTOR INTEGRATION
+# Setting otel-collector-lerian.enabled=true causes the deployment to inject
+# HOST_IP/OTEL_EXPORTER_OTLP_ENDPOINT pointing at the host node's collector.
+# ==============================================================================
+otel-collector-lerian:
+  enabled: false
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)